### PR TITLE
Create command for audition

### PR DESCRIPTION
**Pull request by Evgenij-Kovpak**

### CODE REVIEW

File: Elevel/Application/Features/CommandsForAudition/CreateAuditionCommand.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+using System.Threading.Tasks;
+
+namespace Elevel.Application.Features.CommandsForAudition
+{
+    public class CreateAuditionCommand
+    {
+        public class Request : IRequest<Response>
+        {
+            [JsonIgnore]
+            public Guid CreatorId { get; set; }
```
Comment (parys) [inline Elevel/Application/Features/CommandsForAudition/CreateAuditionCommand.cs:22]: you should populate it inside handler, it should not be in request

File: Elevel/Application/Features/CommandsForAudition/CreateAuditionCommand.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            public long AuditionNumber { get; set; }
+            public string AudioFilePath { get; set; }
```
Comment (parys) [inline Elevel/Application/Features/CommandsForAudition/CreateAuditionCommand.cs:24]: where you get this path?

File: Elevel/Application/Features/CommandsForAudition/UpdateAuditionCommand.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+namespace Elevel.Application.Features.CommandsForAudition
+{
+    
+    public class UpdateAuditionCommand
+    {
+        public class Request : IRequest<Response>
+        {
+            [JsonIgnore]
+            public Guid Id { get; set; }
```
Comment (parys) [inline Elevel/Application/Features/CommandsForAudition/UpdateAuditionCommand.cs:22]: why? client will send id to you

**Comment by andreyrogov8** [comment]
No longer need

**Pull request closed without merging**